### PR TITLE
V1.4 PE-D Bug fixes and UG updates

### LINKED ISSUE #225: [PE-D][Tester E] Discrepancy in sort.
## Description

The screenshot can be explicitly linked to the command above.

## Fix

Same as previous issue.

-------------
Labels: `severity.VeryLow` `type.DocumentationBug`
original: c-wenlong/ped#19

### LINKED ISSUE #231: [PE-D][Tester A] Typographical error by help command
As shown in image.

-------------
Labels: `severity.Low` `type.DocumentationBug`
original: AL-ZT/ped#9

### LINKED ISSUE #240: Update help link for help command to point to user guide
Currently, the link given after executing help command points to home page of project instead of user guide.

### PR DESCRIPTION
**Pull request by ZhiWei1010**
This commit updates the User Guide as well as the Help Window UI.
* Update sortResult screenshot in UG to correctly display sort patients by preferred name from fresh data sample
* Update all command section to specify that screenshot displayed is for executing `sort p` command
* Update example in delete command to match with other example
* Update example in delete event command to work correctly with fresh sample data
* Update helpMessage screenshot in UG to correcty display the updated link in help message
* Update USERGUIDE_URL in UI#helpWindow to correctly display the link to user guide

Fixes #240 Fixes #231 Fixes #225

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -418,7 +418,9 @@ Examples:
 * `sort n`
 * `sort p`
 
-![result for 'sort'](images/sortResult.png)
+This screenshot shows the result of executing `sort p`.
```
Comment (choowengyan) [inline docs/UserGuide.md:445]: To standardise this, may need to check if other command section contains the explanation of screenshot as well.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -81,7 +81,7 @@ command to run the application.<br>
 
 ### Viewing help : `help`
 
-Shows a message explaning how to access the help page.
+Shows a message explaining how to access the help page.
```
Comment (choowengyan) [inline docs/UserGuide.md:84]: Good catch on the spelling error!

**Review by choowengyan — CHANGES_REQUESTED** [review]
Overall, great changes made! Just to check if the explanation is needed for the rest of the doc mentioned.

File: docs/UserGuide.md (hunk 2 above)
Comment (ZhiWei1010) [reply docs/UserGuide.md:84]: It was raised in the PE-D #231 !

File: docs/UserGuide.md (hunk 1 above)
Comment (ZhiWei1010) [reply docs/UserGuide.md:445]: After offline discussion, we decide to standardise and add this line to all other commands!

**Review by choowengyan — APPROVED** [review]
LGTM! Well done on the screenshot description!

**Pull request merged**